apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/db/model/OptionDAO.java

```diff
@@ -99,8 +100,25 @@ public void deleteAll() {
                 int deletedRows = stmt.executeUpdate();
             }
             catch (SQLException e) {
-                LOG.error(e.getMessage());
+                log.error(e.getMessage());
             }
     }
 
+    public boolean exist(String optionKey) {
+        TransactionalDataSource dataSource = databaseManager.getDataSource();
+        try (Connection con = dataSource.getConnection()) {
+            PreparedStatement stmt = con.prepareStatement("SELECT count(*) FROM option WHERE name = ?");
```

Comment:
better to use `select 1 from option` because you want just to check that such record exist